src/components/node-list/node-list-items.js

```diff
@@ -162,8 +162,8 @@ export const getFilteredModularPipelineItems = createSelector(
         selected: false,
         faded: false,
         visible: true,
-        disabled: false,
-        checked: true,
+        disabled: focusMode !== null && focusMode?.id !== modularPipeline.id,
```

Comment:
we need to check for `null` value given that this variable is set as `null` by default, and we need to enforce the first check to ensure that there is a selected focusMode before conducting the second check - i.e without the `focusMode !== null` check then all modular pipelines would be disabled by default when not in focusMode, which is not what we want. ( You can also verify this by taking this test out and you will see what I mean 😀 )

Thanks for raising this nonetheless, this would've been a valid point otherwise